environments/staging/inventory.ini.j2

```diff
@@ -59,9 +59,13 @@ couch0
 [rabbit0]
 {{ rabbit0-staging }} hostname=rabbit0-staging ec2=yes
 
+[rabbit1]
+{{ rabbit1-staging }} hostname=rabbit1-staging ec2=ena ansible_python_interpreter=/usr/bin/python3
```

Comment:
It's now preferred to use `{{ __rabbit1__ }}` syntax, which I added recently and described here: https://github.com/dimagi/commcare-cloud/pull/2812#issuecomment-487168950. I should have made sure to notify you on it, my apologies